### DISCUSSION TITLE
Sending Databricks notebook information in HTTP dispatcher for codeless initialization

### DISCUSSION THREAD

**Discussion by rupesh3020** [AbsaOSS/spline-spark-agent#628] [Q&A] [answered]
Hi Team,

Is there any way to send notebook related information to spline . Currently structure is this:
![image](https://user-images.githubusercontent.com/76835566/226863060-4a3a162d-0a82-45c8-a6b6-c5cd93bc935a.png)

Can we have notebook info in this?

**Comment by wajda** [comment]
Not out of the box, but it can be done with a custom filter. See this post for instance - https://github.com/AbsaOSS/spline-spark-agent/discussions/620#discussioncomment-5215109

**Comment by cerveada** [comment, marked as answer]
There is a new option in Agent 1.0 created for adding user information into lineage JSON.

I just created PR with documentation for it. You can see it here:
https://github.com/AbsaOSS/spline-spark-agent/pull/629/files

It is a bit different from creating the filter yourself. Here you just define what should be added via a json with some specified elements.
